Anabi.DataAccess.Ef/EntityConfigurators/AssetConfig.cs

```diff
@@ -73,11 +73,12 @@ public void SetupEntity(ModelBuilder modelBuilder)
             entity.Property(p => p.LastChangeDate)
                 .HasColumnType("Datetime");
 
+            entity.Property(p => p.Identificator)
+                .HasColumnType("string")
```

Comment:
I think EF automatically infers the type from C# (for properties where you don't have to specify any special attribute, like the precision in the case of a decimal).